.travis.yml

```diff
@@ -9,15 +9,17 @@ cache:
   directories:
   - $HOME/.m2
 
+install: /bin/true
+
 jobs:
   include:
     - stage: build
-      script: mvn -B -V -DskipTests -DperformRelease -Dgpg.skip clean install
+      script: mvn -B -V -DskipTests -DperformRelease -Dgpg.skip clean install -Psnapshots
     - stage: build
-      script: mvn -B -V -DskipTests clean install
+      script: mvn -B -V -DskipTests clean install -Psnapshots
       jdk: openjdk11
     - stage: test
-      script: mvn -B -V test
+      script: mvn -B -V test -Psnapshots
```

Comment:
Maybe we can store the profile in a variable, as we need to change it when the API gets staged or released later.

Reply:
I agree, just wanted to see if it worked first. Will add a variable. Thanks!

Reply:
Great, thanks! :+1: